project.mml

```diff
@@ -200,7 +200,7 @@ Layer:
             CASE WHEN tags->'intermittent' IN ('yes')
               OR tags->'seasonal' IN ('yes', 'spring', 'summer', 'autumn', 'winter', 'wet_season', 'dry_season')
               THEN 'yes' ELSE 'no' END AS int_intermittent,
-            CASE WHEN tunnel IN ('yes', 'culvert') 
+            CASE WHEN tunnel IN ('yes', 'culvert')
```

Comment:
We should not combine this with unrelated whitespace changes, especially not in the road layers because this makes merging with other changes more difficult.

Reply:
Ah good point, this was autoformatting by my IDE. I try not to commit these changes, but this slipped through. I will revert these lines.

Reply:
If you want to do a fixup of all files to comply with .editorconfig as a separate PR that would be welcome.

Reply:
Yes I will do that after this PR is merged :+1:

Reply:
PR in https://github.com/gravitystorm/openstreetmap-carto/pull/4966